spec/controllers/catalog_controller_spec.rb

```diff
@@ -28,11 +26,11 @@
 
   describe 'route check' do
     it 'routes /catalog/:id properly' do
-      expect(get: '/catalog/:id').to route_to(controller: 'catalog', action: 'show', id: ':id')
+      expect(get: '/catalog/id').to route_to(controller: 'catalog', action: 'show', id: 'id')
```

Comment:
did you just change this because it's weird to pass in the string value ':id' as an argument to the catalog controller? or was there something failing?

Reply:
for some reason that I could not figure out exactly why, it started to fail when I had the multiple punct changes. I kept it like this after reverting those changes because of what you mention like a bit weird string value like a symbol.

Reply:
ah of course, it saw the punctuation we were looking for and wigged out. makes sense! thanks. very odd that we ever passed in a symbol-like string here anyway 🤷 

thanks